src/main/resources/application.yml

```diff
@@ -0,0 +1,26 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://192.168.1.50:5433/ops
+    username: postgres
+    password: admin
+  jpa:
+    hibernate:
+      ddl-auto: update
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+        format-sql: true
+    show-sql: false
```

Comment:
setting show-sql property to true in the development phase will be useful, and make false on production to avoid leaking sensitive information.

Reply:
Can you assist me with this?

Reply:
Changes Updated. Kindly review it.